mitxpro/management/commands/configure_instance.py

```diff
@@ -0,0 +1,173 @@
+- createsuperuser to create the super user
+- configure_wagtail for initial setup of Wagtail assets
+- seed_data to creating seed/dummy courses and programs
+
+There are some steps that this command won't do for you:
+- Completing the integration between MITxPro and devstack - there are still
+  steps that you need to take to finish that process
+
+"""
+
+from django.core.management import BaseCommand, call_command
+from oauth2_provider.models import Application
+
+
+class Command(BaseCommand):
+    """
+    Bootstraps a fresh MITxPro instance.
+    """
+
+    def add_arguments(self, parser):
+        """Parses command line arguments."""
+
+        parser.add_argument(
+            "platform",
+            help="Your platform (none, macos, or linux; defaults to none). None skips OAuth2 record creation.",
+            type=str,
+            choices=["none", "macos", "linux"],
+            nargs="?",
+            const="none",
+        )
+
+        parser.add_argument(
+            "--dont-create-superuser",
+            "-S",
+            help="Don't create a superuser account.",
+            action="store_false",
+            dest="superuser",
+        )
+
+        parser.add_argument(
+            "--edx-oauth-client",
+            help="Use the provided OAuth2 client ID, rather than making a new one.",
+            type=str,
+            nargs="?",
+        )
+
+        parser.add_argument(
+            "--edx-oauth-secret",
+            help="Use the provided OAuth2 client secret, rather than making a new one.",
+            type=str,
+            nargs="?",
+        )
+
+        parser.add_argument(
+            "--gateway",
+            help="Specify the gateway IP. (Required for Linux users.)",
+            type=str,
+            nargs="?",
+        )
+
+        parser.add_argument(
+            "--tutor",
+            "-T",
+            help="Configure for Tutor.",
+            action="store_true",
+            dest="tutor",
+        )
+
+        parser.add_argument(
+            "--tutor-dev",
+            help="Configure for Tutor Dev/Nightly.",
+            action="store_true",
+            dest="tutordev",
+        )
+
+    def determine_edx_hostport(self, *args, **kwargs):  # noqa: ARG002
+        """Returns a tuple of the edX host and port depending on what the user's passed in"""
+
+        if kwargs["tutor"]:
+            return ("local.edly.io", "")
+        elif kwargs["tutordev"]:
+            return ("local.edly.io:8000", ":8000")
+        else:
+            return ("edx.odl.local:18000", ":18000")
+
+    def handle(self, *args, **kwargs):  # noqa: ARG002
+        """Coordinates the other commands."""
+
+        (edx_host, edx_gateway_port) = self.determine_edx_hostport(**kwargs)
+
+        # Step 1: run createsuperuesr
+        if kwargs["superuser"]:
+            self.stdout.write(self.style.SUCCESS("Creating superuser..."))
+
+            call_command("createsuperuser")
+
+        # Step 2: create OAuth2 provider records
```

Comment:
Would it be possible to move this step, or at least its output, to the end? The current setup places the logs from other steps after it, which requires users to scroll up to find the client id and secret they need to copy for the next steps in configuring edx.

Reply:
added at the end as well